stays/tests/forms_tests.py

```diff
@@ -0,0 +1,52 @@
+from django.test import TestCase
+from stays.forms import StayForm, ReviewForm
+from customers.tests.factories import PersonFactory, ProviderFactory, PetFactory
+
+
+class StayFormTests(TestCase):
+    def test_owner_is_the_same_person_as_provider(self):
+        owner = PersonFactory.create()
+        provider = ProviderFactory.create(person=owner)
+        pet = PetFactory.create()
+        data = {"owner": owner, "provider": provider, "start_date": "2022-01-01", "end_date": "2022-02-02", "pets": [pet]}
```

Comment:
The date is mostly repeating itself. You can define the setup method where all the necessary objects  used for the tests. Also another method `get_data(self, **kwargs)` should be defined that lets you get the data object you want and makes it easier to change the properties based on the test needs.